### ISSUE TITLE
Sorting produce typescript code which will not compile

### ISSUE THREAD

**Issue by ddeath** [SoominHan/import-sorter#45]
I just noticed if I have imports:

```
import * as GQL from '@gql-schema'
import { someFn } from '@gql-schema'

someFn()
```

it will merge the imports:
```
import * as GQL from '@gql-schema'

someFn()
```

and the result is invalid code (missing import for Fn)

**Comment by mike-co** [comment]
Is there any reason to have:
import { someFn } from '@gql-schema' ? As you have already imported everything using:import * as GQL from '@gql-schema'. Therefore you should be able to access the function by GQL.someFn()

**Comment by ddeath** [comment]
I agree that it is possible to write it like so (i rewrote it in my code) but the point is that I am not expecting extension which is supposed to sort imports to break my code

**Comment by brane53** [comment]
One reason I can think of is for importing types separately from the library code.

Example

```
import * as moment from 'moment';
import { Moment } from 'moment';

let myMoment: Moment = moment();
```
You could technically write:
```
let myMoment: moment.Moment = moment();
```
But I think that looks a lot more messy. Especially if you are using types everywhere.

**Comment by daidodo** [comment]
Issue solved in [this extension](https://marketplace.visualstudio.com/items?itemName=dozerg.tsimportsorter).
